### PR TITLE
Added error-handling for 'appdmg' npm module to our CMake file.

### PR DESCRIPTION
**Pull request by bobsummerwill**
Because Jenkins is just carrying on, even though the El Capitan build machine is missing a correctly installed npm module.

### CODE REVIEW

**Comment by chfast** [comment]
Have anyone reviewed that? This change is a crap.

File: cmake/scripts/appdmg.cmake [hunk 1]
```diff
@@ -13,5 +13,8 @@ configure_file(${APP_DMG_FILE} ${OUTFILE})
 execute_process(COMMAND ${CMAKE_COMMAND} -E copy "${APP_DMG_ICON}" "${ETH_BUILD_DIR}/appdmg_icon.icns")
 execute_process(COMMAND ${CMAKE_COMMAND} -E copy "${APP_DMG_BACKGROUND}" "${ETH_BUILD_DIR}/appdmg_background.png")
 execute_process(COMMAND ${CMAKE_COMMAND} -E remove "${ETH_BUILD_DIR}/Ethereum.dmg") 
-execute_process(COMMAND ${APP_DMG_EXE} ${OUTFILE} "${ETH_BUILD_DIR}/Ethereum.dmg")
+execute_process(COMMAND ${APP_DMG_EXE} ${OUTFILE} "${ETH_BUILD_DIR}/Ethereum.dmg" RESULT_VARIABLE returnCode)
```
Comment (chfast) [inline cmake/scripts/appdmg.cmake:16]: Name variables with capital letters.

File: cmake/scripts/appdmg.cmake [hunk 2]
```diff
@@ -13,5 +13,8 @@ configure_file(${APP_DMG_FILE} ${OUTFILE})
...
 
+if (NOT "${returnCode" STREQUAL "0")
```
Comment (chfast) [inline cmake/scripts/appdmg.cmake:18]: I guessed you missed `}`. Why nobody is writing simple cmake if. It should be:

```
if (NOT returnCode EQUAL 0)
```

**Comment by bobsummerwill** [comment]
Nope - I'm just iterating on these myself.   Will fix now.

**Comment by bobsummerwill** [comment]
My bad.   I tweaked it after a local run.  Dickhead.

**Comment by bobsummerwill** [comment]
I've been running without review on these because there has usually been nobody available to review these CMake edits in my timezones, and I've needed to be iterating between CMake and Jenkins runs.    I'm the only immediate "victim" of bad code.

I've been sharing the merged edits on the gitter channel for post-commit feedback (like yours), so thanks, and please do have a look at the previous ones too.    Basically everything in webthree-helper for the last few days.

**Comment by chfast** [comment]
I have some good experience with _post commit reviews_ but in bigger teams. No idea how to do it right in GitHub.

**Comment by bobsummerwill** [comment]
I think we could probably hook things up so that we get e-mails post-commit for every change.    I'm getting mails for all PRs and all comments on PRs already.    I wouldn't mind getting those too.

**Comment by chfast** [comment]
That's not what I mean. It is called the post commit review in SVN world. In case of git/GitHub that should be PR.

The workflow you use: create PR -> merge it yourself is ok. The issue is nobody is reviewing it, for many reasons:
1. Team is too small. Some changes are unnoticed.
2. Team is not used to review more.
3. We don't have a complete protocol.

**Comment by bobsummerwill** [comment]
Right ... so some things never get reviewed, you mean?

Rather than having a defined process for things being systematically reviewed post-commit?

**Pull request merged**